include/memkind/internal/memkind_memtier.h

```diff
@@ -232,11 +232,8 @@ int memtier_ctl_set(struct memtier_builder *builder, const char *name,
 // float get_obj_hotness(int size);
 
 // PEBS
-// smaller value -> more frequent sampling
-// 10000 = around 100 samples on *my machine* / sec in matmul test
-#define SAMPLE_FREQUENCY 10000
-// #define PEBS_FREQ_HZ     20.0
-#define PEBS_FREQ_HZ     5.0
+extern unsigned long sample_frequency;
+extern float pebs_freq_hz;
```

Comment:
Why float? Unless vectorized or memory-bound, double operations should be as fast as float operations, with minimal memory overhead.